WPMedia/fusion-news-theme-blocks
# Tmedia 729 subheadline

## Description

Update subheadline to always render h2 and use new architecture

## Jira Ticket

- [TMEDIA-729](https://arcpublishing.atlassian.net/browse/TMEDIA-729)

## Acceptance Criteria

Headline Block

Subheadline block is migrated to the new architecture 

Includes Custom fields of:

‘Value to display’ - with dropdown options of Subheadline or Description

## Test Steps

1. Checkout this branch `git checkout TMEDIA-729-subheadline`
2. `npm run storybook`
3. Ensure this matches the heading level 6 documentation in the storybook
![Screen Shot 2022-05-16 at 14 01 54](https://user-images.githubusercontent.com/5950956/168663722-a44faac2-c19f-40a8-a235-32c5a7d10959.png)

## Effect Of Changes

### Before

- uses font-size 20 and 17.6px in live environment for mobile and desktop, respectively

### After

- Now it uses matching heading level 6 based on migration guidance 

## Dependencies or Side Effects

- Other prs for manifest and feature pack 
- https://github.com/WPMedia/arc-themes-manifests/pull/52 https://github.com/WPMedia/arc-themes-feature-pack/pull/244

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._


# Question: 

- There's no matching global font-size for 26px desktop size. What is the desired resolution for these cases? Referenced in ticket -> using header 6 based on design question in other pr
- Removed what looked like test code that leaked in of setting the default value to be "Subheadline". I removed this. This seems like it would be ok, right? We don't have this pattern elsewhere. -> it is needed
